src/components/molecules/Forms/CreateClaim/CreateClaim.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+
+import { Input } from 'components/atoms/Input/Input'
+import { Form } from 'components/molecules/Form/Form'
+
+import './CreateClaim.scss'
+
+interface CreateClaimProps {
+  readonly onSubmit: (event: any) => any
```

Comment:
Can we avoid `any` with `Event`?

for example:

```
readonly onSubmit: (event: Event) => void 
```

is it possible?

Reply:
Ok I updated it.